docker-compose.yml

```diff
@@ -15,17 +15,17 @@ services:
       - 5432:5432
     networks:
       - uszyj-maseczke-network
-  
+
   web:
-    build: ./backend
+    build: ./pomocnicy-online-backend
```

Comment:
Let's use another docker-compose file for the transition period, which overrides `web`

Reply:
We can add another service like `backend`, you can copy-paste web section and change the name/parameters